src/FileIO.scala

```diff
@@ -0,0 +1,53 @@
+import java.io.{File, PrintWriter}
+
+import scala.collection.mutable
+import scala.io.Source
+
+object FileIO {
+
+  val purchases = mutable.MutableList[(String, String, String, String, String)]()
+
+  def main(args: Array[String]) {
+
+    Source
+      .fromFile("be594d3c-purchases.csv")
+      .getLines
+      .drop(1)
+      .foreach( line => {
+        val Array(customer_id, date, credit_card, cvv, category) = line.split(",").map(_.trim)
+        purchases.+=((customer_id, date, credit_card, cvv, category.toUpperCase))
+      })
+
+    def prompt(s: String) = {println(s); io.StdIn.readLine()}
+
+    def menu = {
+      val seq = Seq("alcohol", "furniture", "shoes", "toiletries", "food", "jewelry").mkString("\n")
+      prompt(s"\nPlease select your choice:\n${seq}\n").toUpperCase
+    }
+
+    def sort(s: String) = {
+      val pw = new PrintWriter(new File(s"filtered_${s}_purchases.prn" ))
+      purchases.foreach(x =>
+      if (x._5 == s) {
```

Comment:
since you are essentially doing a filtering here, wouldn't it be simpler to just use the .filter() method:
    purchases.filter(_.5 == s).foreach(x =>